libs/zedUpload/README.md

```diff
@@ -0,0 +1,22 @@
+# zedUpload
+
+This directory contains functions for uploading and downloading to various remote file storage services, specifically:
+
+* AWS S3
+* Azure Blob
+* http/s
+* sftp
+
+## Testing
+
+In order to run full tests, you need to have a remote account. To run the tests:
+
+1. Set an environment variable for the credentials you need. Those are listed below. If a credential is missing for a test, it simply will not run.
+1. Run tests with `go test -tags private_tests .`
```

Comment:
Does this approach mean that the test code will not even be compiled unless it is invoked as above?
Or does it get compiled by a go test from some Makefile (eve/Makefile?) and not run? 
I think compiling the test code makes sense to avoid bit rot, even if they can't usefully test anything without the env variables being set. (Except https tests which seem to use public upload and download servers.)

Reply:
It means it won't even get compiled. 

I get your point about bit rot; I will update these to compile but not run without the right env vars.

Reply:
Done